.hashibot.hcl

```diff
@@ -488,6 +488,7 @@ behavior "regexp_issue_labeler_v2" "service_labels" {
     ],
     "service/s3" = [
```

Comment:
[ask] category correct?

This canonical id is used when cloudfront logs to S3.

https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/AccessLogs.html

Should I move category to 'CloudFront'?

Reply:
Changed to cloudfront_canonical_user_id because there are many canonical_ids for awslogsdelivery, not only for cloudfront